esp-hal-common/src/soc/esp32s3/radio_clocks.rs

```diff
@@ -4,8 +4,9 @@ use crate::system::{RadioClockControl, RadioClockController, RadioPeripherals};
 // 19, 20, 21, 22, 23
```

Comment:
Fun fact: this comment is copied from esp-idf where it is also incorrect (lists 13 bits, while the mask has 12 set)